### PR TITLE
 Fixed argument validation for press automation

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @inikulin, @AlexanderMoskovkin, @VasilyStrelyaev

### CODE REVIEW

File: test/client/fixtures/runner/api/press-test.js [hunk 1]
```diff
@@ -428,4 +428,13 @@ $(document).ready(function () {
 
         document.body.appendChild(iframe);
     });
+
+    asyncTest('Press action should accept only string as argument', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/runner/api/press-test.js]:
```
    asyncTest('Press action should only accept a string as an argument', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

File: test/client/fixtures/runner/api/press-test.js [hunk 2]
```diff
@@ -428,4 +428,13 @@ $(document).ready(function () {
...
+        actionsAPI.press(100);
+        setTimeout(function () {
+            equal(currentErrorType, ERROR_TYPE.incorrectPressActionArgument);
+
+            start();
+        }, 300);
+    });
```
Comment (AlexanderMoskovkin) [inline test/client/fixtures/runner/api/press-test.js]: I think it's better to place this test into [the `parse keys string` module](https://github.com/georgiy-abbasov/testcafe-phoenix/blob/f11cb0ac2fd6b087ad19489d99ce8ce9babe7893/test/client/fixtures/runner/api/press-test.js#L122) after the `press incorrect keys combination` test

File: test/client/fixtures/runner/api/press-test.js [hunk 3]
```diff
@@ -428,4 +428,13 @@ $(document).ready(function () {
 
         document.body.appendChild(iframe);
     });
+
+    asyncTest('Press action should accept only string as argument', function () {
+        actionsAPI.press(100);
+        setTimeout(function () {
```
Comment (AlexanderMoskovkin) [inline test/client/fixtures/runner/api/press-test.js]: add an empty line before this line

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by georgiy-abbasov** [comment]
ping @inikulin

**Comment by inikulin** [comment]
lgtm

**Pull request merged**